src/containers/Favorites/index.jsx

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux'
+
+import Home from '../../components/Home'
+
+import { changeSearchBar, favoriteFetchInitialGifs, favoriteSearchKeyword } from '../../actions'
+
+const mapStateToProps = ({ searchBar, favorites }) => ({
+  searchValue: searchBar.value,
+  giphies: favorites.giphies,
+  keywordFilter: favorites.keyword
+})
+
+const mapDispatchToProps = dispatch => ({
+  onSearchChange: event => dispatch(changeSearchBar(event.target.value)),
+  onSearchSubmit: event => {
+    event.preventDefault()
+    dispatch(favoriteSearchKeyword(event.target.search.value))
+  },
+  initialSetup: () => dispatch(favoriteFetchInitialGifs())
```

Comment:
You may use `bindActionCreators` to bind the `dispatch` function. Also, try to avoid to handle logic in these functions (like preventing the default event handling), for legibility reasons and single responsibility of the functions.